docs/advanced_topics/customisation/page_editing_interface.rst

```diff
@@ -8,7 +8,7 @@ Customising the tabbed interface
 
 .. versionadded:: 1.0
 
-As standard, Wagtail organises panels into three tabs: 'Content', 'Promote' and 'Settings'. Depending on the requirements of your site, you may wish to customise this for specific page types - for example, adding an additional tab for sidebar content. This can be done by specifying an ``edit_handler`` property on the page model. For example:
+As standard, Wagtail organises panels for pages into three tabs: 'Content', 'Promote' and 'Settings'. For snippets Wagtail puts all panels into one page. Depending on the requirements of your site, you may wish to customise this for specific page types or snippets - for example, adding an additional tab for sidebar content. This can be done by specifying an ``edit_handler`` property on the page or snippet model. For example:
```

Comment:
> This can be done by specifying an ``edit_handler`` property on the page or snippet model

"property" should be "attribute" here. "property" in Python means a specific thing, and the example below uses an attribute, not a property. This was incorrect in the previous version of the documentation as well, but it may as well get fixed while you're editing in this area.